docs/BreakingChanges/117 COR_PRF_GC_ROOT_HANDLEs are not being enumerated by profilers.md

```diff
@@ -0,0 +1,33 @@
+## 117: COR_PRF_GC_ROOT_HANDLEs are not being enumerated by profilers
```

Comment:
{
  "Id": "117",
  "Title": "COR_PRF_GC_ROOT_HANDLEs are not being enumerated by profilers",
  "Details": "In the .NET Framework v4.5.1, the profiling API `RootReferences2()` is incorrectly never returning `COR_PRF_GC_ROOT_HANDLE` (they are returned as `COR_PRF_GC_ROOT_OTHER` instead). This issue is fixed beginning in the .NET Framework 4.6.",
  "Suggestion": "This issue has been fixed in the .NET Framework 4.6 and may be addressed by upgrading to that version of the .NET Framework.",
  "Link": null,
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/800872",
  "Notes": "We don't currently have a good mechanism for detecting issues in C++ like this, but I'm including this here for completeness's sake\nSource analyzer status: Not planned. Native code analysis needed",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis